custom_components/solarfocus/strings.json

```
@@ -101,8 +101,73 @@
           "1": "More Log wood can be put on the fire"
         }
       },
+      "pb_message_number" : {
+        "state" : {
+          "0": "Keine Meldung",
+          "1": "Interner Speicher ist ungültig"
+        }
+      },
       "pb_state" : {
         "state" : {
+          "0": "Bereitschaft",
+          "1": "Zündphase",
+          "2": "Pelletsbetrieb",
+          "3": "Kesselsolltemperatur erreicht, Nachlauf",
+          "4": "Nachlauf",
+          "5": "keine Anforderung, Nachlauf",
+          "6": "Brenner ausgeschaltet",
+          "7": "Pelletsvorratsbehälter ist leer, Nachlauf",
+          "8": "Wärmetauscherreinigung ist aktiv",
+          "9": "Wärmetauscherreinigung benötigt, Nachlauf",
+          "10": "Stromausfall, Nachlauf",
+          "11": "zweiter Zündversuch, Nachlauf",
+          "12": "Füllraumtemperatur überschritten, Nachlauf",
+          "13": "Einschub überlastet, Nachlauf",
+          "14": "Fremdkessel aktiv, Nachlauf",
+          "15": "Fremdkessel aktiv, Bereitschaft",
+          "16": "Brenner ausgeschaltet, Nachlauf",
+          "17": "Sicherheitskette ist offen",
+          "18": "Sicherheitskette ist offen, Nachlauf",
+          "19": "Restsauerstoffgehalt zu hoch, Nachlauf",
+          "20": "Abgastemperatur zu gering, Nachlauf",
+          "21": "Abgastemperatur zu hoch, Bereitschaft",
+          "22": "Abgasfühler ist defekt, Bereitschaft",
+          "23": "Einschubtemperatur zu hoch, Bereitschaft",
+          "24": "Lambdasonde ist defekt, Nachlauf",
+          "25": "Kesseltemperatur ist ausreichend, Bereitschaft",
+          "26": "kein Stromfluss Einschubmotor, Nachlauf",
+          "27": "Kesselfühler ist defekt, Bereitschaft",
+          "28": "Kesselfühler ist defekt, Nachlauf",
+          "29": "Lambdasonde wird beheizt",
+          "30": "Wärmetauscherreinigung wartet auf eine Freigabe, Zeit",
+          "31": "Wärmetauscherreinigung wartet auf eine Freigabe, AGT",
+          "32": " Wärmetauscherreinigung wartet auf eine Freigabe, KT",
```

```suggestion
          "32": "Wärmetauscherreinigung wartet auf eine Freigabe, KT",
```
Leading space removed